connect/mirror/src/test/java/org/apache/kafka/connect/mirror/integration/DedicatedMirrorIntegrationTest.java

```diff
@@ -354,8 +355,10 @@ private <T extends SourceConnector> void awaitTaskConfigurations(MirrorMaker mm,
                         .map(TaskInfo::config)
                         .allMatch(predicate);
             } catch (Exception ex) {
```

Comment:
I think this was over-general in the first place, causing the typo. It should be an ExecutionException, and the retriable condition should be simplified. I don't see anything in this method that would throw a synchronous RetriableException.

Reply:
True, I wanted to play it safe but upon reflection I also can't think of a single case where we'd want to handle anything except an ExecutionException that wraps a RebalanceNeededException. I've pushed an update to address this; LMKWYT